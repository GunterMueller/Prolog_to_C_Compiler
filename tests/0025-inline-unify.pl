main :- 123 = 123,
	X = X,
	U = V,
	Y = 123,
	Z = Y, display('1\n'),
	X = Y, display('2\n'),
	X = 123, display('3\n'),
	display([X, Y, Z, U, V]), nl,
	f(X) = f(123).
