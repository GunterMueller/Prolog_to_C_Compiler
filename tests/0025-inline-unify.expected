1
2
3
[123,123,123,_1,_1]
